R/functions_netCDF.R

```diff
@@ -456,42 +654,374 @@ create_netCDF_from_array_with_variables <- function(x, locations, grid,
     }
   }
 
-  if (!has_time_central) {
+  if (!has_T_timeAxis) {
     ncdf4::ncatt_put(nc, varid = 0, attname = "time_label",
       attval = "None")
     ncdf4::ncatt_put(nc, varid = 0, attname = "time_title",
       attval = "No temporal dimensions ... fixed field")
   }
 
-  # add coordinate system attributes
-  prj <- raster::crs(grid)
-  if (!is.na(prj)) {
-    ncdf4::ncatt_put(nc, varid = "crs", attname = "proj4",
-      attval = as.character(prj))
+  # The end --------------------------------------------------------------
+
+  if(verbose == TRUE) {
```

Comment:
`if (verbose)` is sufficient because `verbose` is logical

Reply:
fixed